android/titanium/src/java/org/appcelerator/titanium/proxy/ActionBarProxy.java

```diff
@@ -55,7 +55,12 @@ public ActionBarProxy(AppCompatActivity activity)
 	{
 		super();
 		actionBar = activity.getSupportActionBar();
-		actionBar.setDisplayOptions(ActionBar.DISPLAY_USE_LOGO | ActionBar.DISPLAY_SHOW_HOME | ActionBar.DISPLAY_SHOW_TITLE);
+		// Guard against calls to ActionBar made before inflating the ActionBarView
+		if (actionBar != null) {
+			actionBar.setDisplayOptions(ActionBar.DISPLAY_USE_LOGO | ActionBar.DISPLAY_SHOW_HOME | ActionBar.DISPLAY_SHOW_TITLE);
+		} else {
+			Log.w(TAG, "Trying to get a reference to ActionBar before it's container was inflated.");
```

Comment:
Minor Issue:
The log message's word "it's" should be "its" without an apostrophe.

Reply:
Updated.